webservices/rulemaking_docs/__init__.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import sys
+
+from .rulemaking import load_rulemaking
+
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+logger = logging.getLogger("elasticsearch")
+logger.setLevel("WARN")
```

Comment:
@fec-jli It looks like lines 9 and 11 are duplicates. Could you keep only one of them?